### ISSUE TITLE
Don't use android:allowBackup="true" in the library AndroidManifest

### ISSUE THREAD

**Issue by filipmaelbrancke** [medyo/android-about-page#6]
Consider not setting allowBackup=true in the library.
An app that uses the library, but which has allowBackup=false in its own manifest will get a 'manifest merger failed' message, like
```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:16:9-36
  	is also present at [com.github.medyo:android-about-page:1.0] AndroidManifest.xml:12:9-35 value=(true).
  	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:14:5-66:19 to override.
````

I don't think an 'about page library' should set an allowBackup property

**Comment by medyo** [comment]
True, I didn't touched the `manifest` file since creation ! 
I'll handle that, thanks @filipmaelbrancke

**Comment by medyo** [comment]
It's now fixed, b678c6b
